### PR TITLE
ASQ v7: configuration

### PR DESCRIPTION
**Pull request by Scooletz**
Connects to https://github.com/Particular/PlatformDevelopment/issues/723

### CODE REVIEW

**Comment by Scooletz** [comment]
Ping @Particular/docs-maintainers

**Comment by SimonCropp** [comment]
Should this still have a WIP?

File: nservicebus/azure-storage-queues/configuration.md [hunk 1]
```diff
@@ -6,11 +6,7 @@ tags:
 - Cloud
 ---
 
-The default settings can be overridden by adding a configuration section called `AzureServiceBusQueueConfig` to the web.config or app.config files:
-
-snippet:AzureStorageQueueConfig
```
Comment (mikeminutillo) [inline nservicebus/azure-storage-queues/configuration.md]: Is this the snippet that was deleted above? It has a different name

File: nservicebus/azure-storage-queues/configuration.md [hunk 2]
```diff
@@ -6,11 +6,7 @@ tags:
...
-
-The following values can be modified using this configuration setting:
+Azure Storage Queues Transport can be configured using following parameters. Depending on the version, parameters' values can  be configured using a custom configuration section or code-first approach. Please find the description of the parameters below:
```
Comment (mikeminutillo) [inline nservicebus/azure-storage-queues/configuration.md]: I would keep the first sentence and move the second sentence to after the list. The 3rd sentence is then not needed.

Comment (mikeminutillo) [inline nservicebus/azure-storage-queues/configuration.md]: Should this start with "The"?

> **The** Azure Storage Queues Transport...

Comment (mikeminutillo) [inline nservicebus/azure-storage-queues/configuration.md]: another "the"

> ... can be configured using **the** following parameters.

File: nservicebus/azure-storage-queues/configuration.md [hunk 3]
```diff
@@ -20,3 +16,15 @@ The following values can be modified using this configuration setting:
  * `BatchSize`: The number of messages that the transport tries to pull at once from the storage queue. Defaults to 10. Depending on the expected load, I would vary this value between 1 and 1000 (which is the limit)
 
 NOTE: `QueueName` and `QueuePerInstance` are obsoleted. Instead, use bus configuration object to specify endpoint name and scale-out option.
+
+### Via the App.Config
+
+In Versions 5.0 to 6.x the default settings can be overridden by adding a configuration section called `AzureServiceBusQueueConfig` to the web.config or app.config files:
```
Comment (mikeminutillo) [inline nservicebus/azure-storage-queues/configuration.md]: We usually include minor version numbers unless they are relevant. i.e. 

> In Version 5 and Version 6 of the Transport the default settings...

File: nservicebus/azure-storage-queues/configuration.md (hunk 1 above)
Comment (Scooletz) [inline nservicebus/azure-storage-queues/configuration.md]: No. The snippet is still valid up to version 6. For Version 7 we use code only configuration. I wanted to kill two birds with one stone and provide a generic description first, later, differentiate between the configuration section and the code only configuration. Does it answer your question?

**Comment by Scooletz** [comment]
Ping @Particular/docs-maintainers

**Pull request merged**